CLAUDE.md

```
@@ -1,58 +1,92 @@
 
-## Quick Reference
+# Run clippy lints
+cargo clippy --all
+
+# Format code
+cargo fmt --all
+
+# Build gRPC stubs (requires buf)
+buf generate
+
+# Run the Python MCP server
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)